inventory/hosts

```diff
@@ -207,6 +207,7 @@ uk_prod
 de_prod
 ie_prod
 hu_prod
+fi_prod
```

Comment:
Sorry, I'm just noticing that the `europe` group is used in our release deployments. But we don't have access to your server. So this will fail when we try to deploy. I'll probably remove this entry then.